metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1125,13 +1125,11 @@ private Errors validateAlterPartitionData(
             return INVALID_REQUEST;
         }
 
-        List<Integer> ineligibleReplicas = partitionData.newIsr().stream()
-            .filter(replica -> !isEligibleReplica.apply(replica))
```

Comment:
nit: I think that we could remove `isEligibleReplica` from the function params. It is not used anymore.

Reply:
Thanks for the reminder. Forgot about that.